### ISSUE TITLE
-n arg with nerdctl cp doesn't work

### ISSUE THREAD

**Issue by vsaisujeeth** [containerd/nerdctl#1334]
### Description

namespace arg doesn't work with nerdctl cp cmd. I had to set CONTAINERD_NAMESPACE env variable to overcome this issue.

### Describe the results you received and expected

-n should change namespace

### What version of nerdctl are you using?

latest

**Comment by junnplus** [comment]
```shell
$ sudo nerdctl -n test cp Makefile nginx:/home
FATA[0000] 1 errors: [no such object: nginx]
FATA[0000] failed to execute [/usr/local/bin/nerdctl container inspect --mode=native --format={{json .Process}} nginx]: exit status 1
```

**Comment by fahedouch** [comment]
it is a desired behaviour because `-n is nonPersistentAlias (conflicts with nerdctl logs -n)`  : https://github.com/containerd/nerdctl/blob/master/cmd/nerdctl/main.go#L144 . 
plz use : `nerdctl --namespace test cp Makefile nginx:/home`

**Comment by AkihiroSuda** [comment]
> it is a desired behaviour because -n is nonPersistentAlias (conflicts with nerdctl logs -n)

Still shouldn't conflict with cp

**Comment by fahedouch** [comment]
@AkihiroSuda hmm right is should only conflict with `logs` . I will fix this

**Comment by yardenshoham** [comment]
Is this closed since #1375 was merged?
